receiver/mongodbreceiver/metadata.yaml

```
@@ -0,0 +1 @@
+name: mongodbreceiver
```

```suggestion
name: mongodbreceiver

```
Newline is always required.